lib/Service/ObjectService.php

```diff
@@ -310,7 +310,7 @@ public function getAllObjects(string $objectType, ?int $limit = null, ?int $offs
 	 * @return mixed The created or updated object.
 	 * @throws ContainerExceptionInterface|DoesNotExistException|MultipleObjectsReturnedException|NotFoundExceptionInterface
 	 */
-	public function saveObject(string $objectType, array $object, bool $updateVersion = true): mixed
+	public function saveObject(string $objectType, array $object, array $extend = [], bool $updateVersion = true): mixed
```

Comment:
You did not check where this function is being used, this change will break the following use of this saveObject function:
![image](https://github.com/user-attachments/assets/5eda6593-a379-4cc6-a8d5-5581848e8f99)

Reply:
Fixed in https://github.com/ConductionNL/opencatalogi/pull/167/commits/6b14d21af779b039423c0e2a5dde21b96d37abd7